com.unity.netcode.gameobjects/Tests/Editor/UI/UITestHelpers.cs

```diff
@@ -0,0 +1,14 @@
+﻿using UnityEditor;
+using UnityEngine;
+
+namespace Unity.Netcode.EditorTests
+{
+    static class UITestHelpers
+    {
+        [MenuItem("Netcode/UI/Reset Multiplayer Tools Tip Status")]
+        static void ResetMultiplayerToolsTipStatus()
+        {
+            PlayerPrefs.DeleteKey("NGO_Tools_Tip_Dismissed");
```

Comment:
This should probably refer to the same constant 🤔

Reply:
and maybe make the constant accessible internally to use it in both places :)

Reply:
I originally didn't do this because I didn't think the test assembly referenced the editor assembly yet. Looks like I was wrong. I'll need to expose internals but not a big deal